identities/models.py

```diff
@@ -44,3 +44,11 @@ class Identity(models.Model):
 
     def __str__(self):
         return str(self.id)
+
+
+class OptOut(models.Model):
+    """An opt_out"""
+    identity = models.ForeignKey(Identity)
```

Comment:
I think this should be nullable, we want to be able to get either an identity or a address type and address pairing that we convert into a identity in the post_save task